# Reading List


## scATAC-seq methodology papers
- [ArchR](https://www.biorxiv.org/content/10.1101/2020.04.28.066498v1), "ArchR: An integrative and scalable software package for single-cell chromatin accessibility analysis"
- [Signac](https://www.biorxiv.org/content/10.1101/2020.11.09.373613v1), "Multimodal single-cell chromatin analysis with Signac"
- [cisTopic](https://www.nature.com/articles/s41592-019-0367-1), "cisTopic: cis-regulatory topic modeling on single-cell ATAC-seq data"
- [SnapATAC](https://www.biorxiv.org/content/10.1101/615179v3), "SnapATAC: A Comprehensive Analysis Package for Single Cell ATAC-seq"


## scATAC-seq tutorials
- [ArchR brief tutorial](https://www.archrproject.com/articles/Articles/tutorial.html), "A Brief Tutorial of ArchR"
- [ArchR full tutorial](https://www.archrproject.com/bookdown/index.html), "ArchR: Robust and scaleable analysis of single-cell chromatin accessibility data."
- [Signac tutorial](https://satijalab.org/signac/articles/pbmc_vignette.html), "Analyzing PBMC scATAC-seq with Signac"
- [cisTopic tutorial](http://htmlpreview.github.io/?https://github.com/aertslab/cisTopic/blob/master/vignettes/WarpLDA_10X_workflow.html), "cisTopic on 10X 5k PBMCs data set"


## scATAC-seq review
- [Assessment of computational methods for the analysis of single-cell ATAC-seq data](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-019-1854-5)
- [Single-cell ATAC sequencing analysis: From data preprocessing to hypothesis generation](https://www.sciencedirect.com/science/article/pii/S2001037020303019)


## sc multiomics
- [Multiomics analysis reveals extensive epigenome remodeling during cortical development](https://www.biorxiv.org/content/10.1101/2020.08.07.241828v1)
- [CellOracle - Dissecting cell identity via network inference and in silico gene perturbation](https://www.biorxiv.org/content/10.1101/2020.02.17.947416v3)


## Epigenetics, regulatory genomics
- [A pitfall for machine learning methods aiming to predict across cell types](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02177-y)
- [DNA mismatches reveal conformational penalties in proteinâ€“DNA recognition](https://www.nature.com/articles/s41586-020-2843-2)
- [Regulation of single-cell genome organization into TADs and chromatin nanodomains](https://www.nature.com/articles/s41588-020-00716-8#Sec3)
- [Deep Learning library for enhancing Single Cell Analysis](https://medium.com/bayes-labs/deep-learning-library-for-enhancing-single-cell-transcriptome-data-analysis-part-1-74d3cfcc8937)
- [ATAC-seq normalization method can significantly affect differential accessibility analysis and interpretation](https://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-020-00342-y)


## general
- [Science Forum: Ten common statistical mistakes to watch out for when writing or reviewing a manuscript](https://elifesciences.org/articles/48175)
- [single cell omics Germany - database](https://www.singlecell.de/index.php/resources/databases/)


## Other resources
- [awesome-single-cell github page](https://github.com/seandavi/awesome-single-cell)

About single-cell
- [Single Cell OMICS Germany (SCOG)](https://www.singlecell.de)
- [Single Cell Genomics Day (Satija Lab)](https://satijalab.org/scgd/)
- [Bioinformatics for Benched Biologists (Lior Patcher)](https://www.youtube.com/watch?v=vKsh_ovq8x8)
- [Bioinformatics for Benched Biologists II (R edition - Lior Patcher)](https://www.youtube.com/watch?v=ALOYyOEzORg)
- [OzSingleCell Consortium](https://www.singlecells.org.au)

About epigenetics, regulatory genomics
- [San Diego Chromatin Club](https://medschool.ucsd.edu/som/cmm/research/epigenomics/education-outreach/Pages/Seminars.aspx)
- [Fragile Nucleosome](https://generegulation.org/fragile-nucleosome/)

Comaparison between tSNE and UMAP
- [The art of using t-SNE for single-cell transcriptomics](https://www.nature.com/articles/s41467-019-13056-x)

DR
- [Ten quick tips for effective dimensionality reduction](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1006907)

General
- [Tutorial collection from ISCB](https://www.iscb.org/covid19-tutorials)

